sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,23 @@
+UPDATE `creature_template` SET `ScriptName`='npc_hakkari_bloodkeeper' WHERE `entry`=8438;
+UPDATE `creature_template` SET `ScriptName`='npc_nightmare_suppressor' WHERE `entry`=8497;
+UPDATE `creature_template` SET `ScriptName`='boss_avatar_of_hakkar' WHERE `entry`=8443;
+
+DELETE FROM `creature_text` WHERE `CreatureID`=8497;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(8497, 0, 0, 'Stop!  Infidels!', 14, 0, 100, 0, 0, 0, 4532, 0, 'Nightmare Suppressor'),
+(8497, 0, 1, 'You must not summon our god!', 14, 0, 100, 0, 0, 0, 4530, 0, 'Nightmare Suppressor'),
+(8497, 0, 2, 'No!  We cannot allow you to summon Hakkar!', 14, 0, 100, 0, 0, 0, 4529, 0, 'Nightmare Suppressor'),
+(8497, 0, 3, 'No!  You must not do this!', 14, 0, 100, 0, 0, 0, 4531, 0, 'Nightmare Suppressor');
+
+DELETE FROM `creature_text` WHERE `CreatureID`=8440;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES 
+(8440, 0, 0, 'I DRAW CLOSER TO THIS WORLD!', 14, 0, 100, 0, 0, 0, 4543, 0, 'Shade of Hakkar'),
+(8440, 1, 0, 'I TASTE THE BLOOD OF LIFE!', 14, 0, 100, 0, 0, 0, 6252, 0, 'Shade of Hakkar'),
+(8440, 2, 0, 'I AM NEAR!', 14, 0, 100, 0, 0, 0, 6253, 0, 'Shade of Hakkar'),
+(8440, 3, 0, 'HAKKAR LIVES!', 14, 0, 100, 0, 0, 0, 6254, 0, 'Shade of Hakkar');
+
+DELETE FROM `creature_text` WHERE `CreatureID`=8443 AND `GroupID`=0 AND `ID`=0;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES 
+(8443, 0, 0, 'I AM HERE!', 14, 0, 100, 0, 0, 0, 4545, 0, 'Avatar of Hakkar'),
+(8443, 1, 0, 'DIE, MORTALS!', 14, 0, 100, 0, 0, 0, 4546, 0, 'Avatar of Hakkar');
```

Comment:
Based on TDB 335.24011 content, you could select only `CreatureID` = 8443 in the `DELETE FROM` line, seeing that there is currently no `creature_text` for creature 8443. On the other hand, if you need to be specific, just in case there could be other text lines (custom?) for creature 8443, you should include the IDs for both of the lines you insert, not just one of them. Here are my 2 suggestions: 1) generalistic approach, 2) detailed approach
```sql
DELETE FROM `creature_text` WHERE `CreatureID`=8443;
```
```sql
DELETE FROM `creature_text` WHERE `CreatureID`=8443 AND `GroupID` IN (0, 1) AND `ID`=0;
```